omero/developers/build-system.txt

```diff
@@ -248,3 +248,49 @@ it is put, and what role it plays in the distribution.
     By default, |OmeroCpp| are not built. Use ``build-all`` for that.
 
 These files are then zipped to OMERO.server-<version>.zip via ``release-zip``
+
+Coupled development
+-------------------
+
+Since OMERO 5.1.3, Bio-Formats is decoupled from the OMERO build system who
```

Comment:
I think of OMERO as a `which` rather than `who`, but considering Ms. Fair Boot and all ... :smile: